src/crystal/compiler_rt/udivti3.cr

```diff
@@ -0,0 +1,6 @@
+# Functions for returning the product of signed multiplication with overflow eg. `a * b`
+
+fun __udivti3(a : UInt128, b : UInt128) : UInt128
+  r = 0_u128
+  __udivmodti4(a, b, pointerof(r))
+end
```

Comment:
```suggestion
  r
end
```

Reply:
r is the remainder and this is a `div` function not a `mod` function so I believe r should not be returned.